go/multiorch/store/pooler_info.go

```diff
@@ -17,19 +17,27 @@ package store
 import (
 	"time"
 
-	"github.com/multigres/multigres/go/pb/clustermetadata"
+	clustermetadatapb "github.com/multigres/multigres/go/pb/clustermetadata"
 	multipoolermanagerdatapb "github.com/multigres/multigres/go/pb/multipoolermanagerdata"
 )
 
 // PoolerHealth represents runtime state of a MultiPooler instance.
 // This stores:
-// - The MultiPooler record from topology
+// - Core pooler metadata (flattened from MultiPooler proto)
 // - Timestamps for staleness detection
 // - Flattened health metrics from Status RPC
 // - Computed fields for quick access
 type PoolerHealth struct {
-	// MultiPooler record from topology service
-	MultiPooler *clustermetadata.MultiPooler
+	// Core pooler metadata (from topology)
+	ID             *clustermetadatapb.ID
+	Database       string
+	TableGroup     string
+	Shard          string
+	KeyRange       *clustermetadatapb.KeyRange
+	Hostname       string
+	PortMap        map[string]int32
+	TopoPoolerType clustermetadatapb.PoolerType          // Topology type (PRIMARY or REPLICA)
+	ServingStatus  clustermetadatapb.PoolerServingStatus // Current serving status
```

Comment:
Move lines 47-49 here to keep everything other than the core pooler fields alphabetical?